test/it/parser/src/main/resources/case/dml/select-special-function.xml

```diff
@@ -125,10 +125,26 @@
         </projections>
     </select>
     <select sql-case-id="select_char">
-        <projections start-index="7" stop-index="29">
-            <expression-projection text="CHAR(77,121,83,81,'76')" start-index="7" stop-index="29">
+        <projections start-index="7" stop-index="44">
```

Comment:
Why modify `stop-index` to `44`? The projection stop index in sql `SELECT CHAR(77,121,83,81,'76')` is 29, you should keep it with correct value.

Reply:
I optimised it. Thanks alot :)